hoot-core/src/main/cpp/hoot/core/io/HootApiDbWriter.h

```diff
@@ -44,7 +44,7 @@ class HootApiDbWriter : public PartialOsmMapWriter, public Configurable
 public:
 
   static std::string className() { return "hoot::HootApiDbWriter"; }
-  static QString emailKey() { return "hootapi.db.writer.email"; }
+  static QString emailKey() { return "api.db.email"; }
```

Comment:
Again the key exists in the `ConfigOptions` function `static QString getApiDbEmailKey() { return "api.db.email"; }`